searxng/searxng
# [feat] images: show resolution in overview

## What does this PR do?
- the previous `img_format` will now be used for the real format of the image (e.g. jpg, png, webp, ...)
- there now is a new field called `resolution` that should be used with the format `width x height`
- I've also changed the documentation accordingly

## Why is this change important?
see #3177

## How to test this PR locally?
- search for images, and notice the resolution labels next to the source (only works for the here modified engines)

## Related issues
closes #3177
